DotNetWebApi/Models/PicnicReturn.cs

```diff
@@ -0,0 +1,40 @@
+using DotNetWebApi.Models;
+
+public class PicnicReturn 
```

Comment:
When you add an endpoint for getting TeddyBears, you will also want to create a `TeddyBearReturn` class.

Reply:
In addition, ensure references to other models in the `TeddyBearReturn` model are also return classes.